src/components/Delegation/DelegatedToUserCard.tsx

```diff
@@ -1,26 +1,28 @@
+import React, { useMemo, useState } from 'react'
+
 import useFormatMessage from 'decentraland-gatsby/dist/hooks/useFormatMessage'
 import { Button } from 'decentraland-ui/dist/components/Button/Button'
 import { Card } from 'decentraland-ui/dist/components/Card/Card'
 import { Header } from 'decentraland-ui/dist/components/Header/Header'
 import { Loader } from 'decentraland-ui/dist/components/Loader/Loader'
 import { Stats } from 'decentraland-ui/dist/components/Stats/Stats'
-import React, { useMemo, useState } from 'react'
-import { Scores } from '../../entities/Votes/utils'
-import { DelegationResult } from '../../hooks/useDelegation'
+
+import { DelegationResult, ScoreDetail } from '../../api/Snapshot'
 import Empty from '../Common/Empty'
 import Scale from '../Icon/Scale'
 import ActionableLayout from '../Layout/ActionableLayout'
 import VotingPower from '../Token/VotingPower'
-import { VotingPowerListModal } from './VotingPowerListModal'
-import VotingPowerListItem from './VotingPowerListItem'
+
 import './DelegatedToUserCard.css'
+import VotingPowerListItem from './VotingPowerListItem'
+import { VotingPowerListModal } from './VotingPowerListModal'
 
 const DISPLAYED_DELEGATIONS = 5
 const OPEN_CALL_FOR_DELEGATES_LINK = 'https://forum.decentraland.org/t/open-call-for-delegates-apply-now/5840/5'
 
 interface DelegatedToUserCardProps {
   delegation: DelegationResult
-  scores: Scores | null
+  scores: Record<string, ScoreDetail>
```

Comment:
Maybe we should change `Votes/utils.ts` from
`export type Scores = Record<string, number>`
to
`export type Scores = Record<string, ScoreDetail>`